src/addEditErase.js

```diff
@@ -0,0 +1,61 @@
+import { save, load } from './data.js';
+
+const theBigList = document.querySelector('.todos');
+
+export function antiShowAll(element) {
+  while (element.lastElementChild) {
+    element.removeChild(element.lastElementChild);
+  }
+}
+
+export function elimanateCompleteds() {
+  const actualtodo = load();
+  const result = actualtodo.filter((activity) => (activity.completed === false));
+  for (let i = 0; i < result.length; i += 1) {
+    result[i].index = i;
+  }
+  save(result);
+}
+
+export function addActivity(text) {
+  const actualtodo = load();
+  const { length } = actualtodo;
+  const newActivity = {
+    description: text,
+    completed: false,
+    id: Date(),
+    index: length,
+  };
+  actualtodo[length] = newActivity;
+  save(actualtodo);
+}
```

Comment:
- [ ] Great job with your project 👍 . Please make sure you create a test file for each of the following functions:
- a function for editing the task description.
- a function for updating an item's 'completed' status.
- the "Clear all completed" function.

For example: `edit.test.js`